swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+class Clothing(Item):
+
+    def __init__(self, category="", condition=0.0, age=0):
+        super().__init__(category, condition, age)
+        self.category = "Clothing"
```

Comment:
Instead of having an ignored `category` parameter it would be better to pass `"Clothing"` directly into the parent's `__init__`:

```suggestion
    def __init__(self, condition=0.0, age=0):
        super().__init__("Clothing", condition, age)
```